### PR TITLE
add legend-studio-preset-external-format-json-schema

### PR DESCRIPTION
**Pull request by MauricioUyaguari**
| Q                       | A                                |
| ----------------------- | -------------------------------- |
| Fixed issues?           |  |
| Tests added + passed?   | Yes                              |
| Changeset added?        | Yes                              |
| Documentation PR link   |                                  |
| Any dependency changes? | Yes                               |

### CODE REVIEW

File: .changeset/orange-jeans-turn.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-studio-app': patch
+'@finos/legend-studio-preset-external-format-json-schema': patch
+---
+
+Add peset external format- JSON schema
```
Comment (akphi) [inline .changeset/orange-jeans-turn.md:6]: spelling

File: package.json [hunk 2]
```diff
@@ -71,6 +71,9 @@
   "browserslist": [
     "last 2 Chrome versions"
   ],
+  "dependencies": {
```
Comment (akphi) [inline package.json:74]: ??

File: packages/legend-studio-preset-external-format-json-schema/.npmignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+build
```
Comment (akphi) [inline packages/legend-studio-preset-external-format-json-schema/.npmignore:1]: make sure you follow what we have outside. This content is wrong I think

File: packages/legend-studio-preset-external-format-json-schema/_package.config.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+/**
```
Comment (akphi) [inline packages/legend-studio-preset-external-format-json-schema/_package.config.js:1]: Update copyright headers

File: yarn.lock [hunk 5]
```diff
@@ -4637,6 +4654,16 @@ __metadata:
   languageName: node
   linkType: hard
 
+"changeset@npm:^0.2.6":
```
Comment (akphi) [inline yarn.lock:4657]: Don't think any of these changes should happen in `yarn.lock` due to introduction of JSON Schema.

File: packages/legend-studio-app/src/index.tsx [hunk 6]
```diff
@@ -16,12 +16,13 @@
 
 import { Studio } from '@finos/legend-studio';
 import { DSLText_Preset } from '@finos/legend-studio-preset-dsl-text';
+import { EFJSONSchema_Preset } from '@finos/legend-studio-preset-external-format-json-schema';
```
Comment (akphi) [inline packages/legend-studio-app/src/index.tsx:19]: need to add json-schema package to the `pacakge.json` and `tsconfig.json` of this package.

File: packages/legend-studio-preset-external-format-json-schema/package.json [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    "legend",
+    "legend-studio",
+    "studio",
+    "external-format",
+    "external-format-json-schema",
+    "preset",
+    "preset-external-format-json-schema",
+    "json-schema"
+  ],
+  "homepage": "https://github.com/finos/legend-studio/tree/master/packages/legend-studio-preset-external-format-json-schemat",
```
Comment (akphi) [inline packages/legend-studio-preset-external-format-json-schema/package.json:16]: typo `schema`

File: packages/legend-studio-app/src/index.tsx [hunk 8]
```diff
@@ -16,12 +16,13 @@
...
 import studioConfig from '../studio.config';
 import './index.scss';
 
 Studio.create()
   .setup({ baseUrl: studioConfig.baseUrl })
-  .withPresets([new DSLText_Preset()])
+  .withPresets([new DSLText_Preset(), new EFJSONSchema_Preset()])
```
Comment (akphi) [inline packages/legend-studio-app/src/index.tsx:25]: oh how I hate the fact this must be a separate module in Studio 😡 😡 😡

Comment (MauricioUyaguari) [reply packages/legend-studio-app/src/index.tsx:25]: okay sir

**Review by akphi — APPROVED** [review]

**Pull request merged**